### ISSUE TITLE
Accept multiple axes in `.interp()`

### ISSUE THREAD

**Issue by jbusecke** [xgcm/xgcm#157]
I often find myself interpolating variables from the B-grid velocity point (north-east) to the tracer point, or vice versa. 
Currently this looks like this:
```
da_new = grid.interp(grid.interp(da, 'X'), 'Y')
```
I would love to be able to pass a list of axes to achive the same behavior like this:
```
da_new = grid.interp(da, ['X', 'Z'])
```

This would be a nice addition to have also for the `min()` and `max()` functions. A use case for this would be to determine the cell depth of z-star cells in MOM5, which at the velocity points are defined as the minimum of the 4 surrounding tracer cells.

**Comment by dcherian** [comment]
+20.

**Comment by rabernat** [comment]
Great idea.

Implementation question: does the order matter here?

**Comment by dcherian** [comment]
Hmm it's a bunch of linear operations so order shouldn't matter?

**Comment by jbusecke** [comment]
Hmm I have always assumed that not, but perhaps there is a special case. 
We could still accomodate that by internally following the order of the list. E.g. ‘grid.interp(..., [‘X’, ‘Y’])’ vs ‘grid.interp(..., [‘Y’, ‘X’])’
What do you think?

Julius J.M. Busecke, Ph.D. (he/him)
———————————————
Postdoctoral Research Associate
Princeton University • Geosciences
408A Guyot Hall • Princeton NJ
juliusbusecke.com

**Comment by jbusecke** [comment]
I just put out an idea how to implement this in #159, feedback greatly appreciated.

**Comment by dcherian** [comment]
THis can be closed?

**Comment by jbusecke** [comment]
Yes. Closed via #159
